baseplate/lint/cql_string_format_plugin.py

```diff
@@ -0,0 +1,89 @@
+from astroid import nodes
+from pylint.checkers import BaseChecker
+from pylint.interfaces import IAstroidChecker
+from pylint.lint import PyLinter
+
+
+class NoCQLStringFormatChecker(BaseChecker):
+    __implements__ = IAstroidChecker
+
+    name = "no-cql-string-format"
+    priority = -1
+    msgs = {
+        "W0001": (
+            "Python string formatting found in Cassandra database query",
```

Comment:
Would this also fire if someone does this with a SQLAlchemy query? I think that's a good thing but we may want to update the messages to not be Cassandra-specific in that context.